### PR TITLE
Wrong link

### PR DESCRIPTION
**Pull request by Dyrimon**
Resolves: #1164

### CODE REVIEW

File: docs/productivity.en.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Get working and collaborating without sharing your documents with a middleman or
 
         - [:fontawesome-brands-windows: Windows](https://www.onlyoffice.com/download-desktop.aspx?from=default)
```
Comment (lx-is) [inline docs/productivity.en.md:44]:
```suggestion
        - [:fontawesome-brands-windows: Windows](https://www.onlyoffice.com/download-desktop.aspx)
```

File: docs/productivity.en.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Get working and collaborating without sharing your documents with a middleman or
...
         - [:fontawesome-brands-apple: macOS](https://www.onlyoffice.com/download-desktop.aspx?from=default)
```
Comment (lx-is) [inline docs/productivity.en.md:45]:
```suggestion
        - [:fontawesome-brands-apple: macOS](https://www.onlyoffice.com/download-desktop.aspx)
```

File: docs/productivity.en.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Get working and collaborating without sharing your documents with a middleman or
...
-        - [:fontawesome-brands-linux: Linux](https://www.libreoffice.org/download/download/)
+        - [:fontawesome-brands-linux: Linux](https://www.onlyoffice.com/download-desktop.aspx?from=default)
```
Comment (lx-is) [inline docs/productivity.en.md:46]:
```suggestion
        - [:fontawesome-brands-linux: Linux](https://www.onlyoffice.com/download-desktop.aspx)
```

**Review by lx-is — CHANGES_REQUESTED** [review]
Might as well clean up all the links

**Comment by dngray** [comment]
Oh I didn't see this PR, so I created https://github.com/privacyguides/privacyguides.org/pull/1168 fixed in https://github.com/privacyguides/privacyguides.org/commit/cebf65a89b19325725e445b11a700bb3fc5c6851

**Pull request closed without merging**